packages/angular_devkit/build_angular/test/browser/scripts-array_spec_large.ts

```diff
@@ -11,19 +11,18 @@ import { TestLogger } from '@angular-devkit/architect/testing';
 import { logging } from '@angular-devkit/core';
 import { browserBuild, createArchitect, host } from '../utils';
 
-
 describe('Browser Builder scripts array', () => {
   const scripts: { [path: string]: string } = {
-    'src/input-script.js': 'console.log(\'input-script\'); var number = 1+1;',
-    'src/zinput-script.js': 'console.log(\'zinput-script\');',
-    'src/finput-script.js': 'console.log(\'finput-script\');',
-    'src/uinput-script.js': 'console.log(\'uinput-script\');',
-    'src/binput-script.js': 'console.log(\'binput-script\');',
-    'src/ainput-script.js': 'console.log(\'ainput-script\');',
-    'src/cinput-script.js': 'console.log(\'cinput-script\');',
-    'src/lazy-script.js': 'console.log(\'lazy-script\');',
-    'src/pre-rename-script.js': 'console.log(\'pre-rename-script\');',
-    'src/pre-rename-lazy-script.js': 'console.log(\'pre-rename-lazy-script\');',
+    'src/input-script.js': "console.log('input-script'); var number = 1+1;",
```

Comment:
Surprised the use of `"` does not break our linter.

Reply:
That is weird indeed.